### DISCUSSION TITLE
2024/08/02/%E6%88%91%E6%98%AF%E5%A6%82%E4%BD%95%E7%AE%A1%E7%90%86%E8%87%AA%E5%B7%B1%E7%9A%84%E6%9C%8D%E5%8A%A1%E5%99%A8%E7%9A%84/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#6] [Announcements]
# 2024/08/02/%E6%88%91%E6%98%AF%E5%A6%82%E4%BD%95%E7%AE%A1%E7%90%86%E8%87%AA%E5%B7%B1%E7%9A%84%E6%9C%8D%E5%8A%A1%E5%99%A8%E7%9A%84/

同时拥有多台云服务器，如何管理是好？

https://blog.byteloid.one/2024/08/02/%E6%88%91%E6%98%AF%E5%A6%82%E4%BD%95%E7%AE%A1%E7%90%86%E8%87%AA%E5%B7%B1%E7%9A%84%E6%9C%8D%E5%8A%A1%E5%99%A8%E7%9A%84/

**Comment by ltzXiaoYanMo** [comment]
我：

服务商-地区-可用区

列如：<br>
TencentCloud-BJ-3（腾讯云北京三区）

AlibabaCloud-HZ-J（阿里云杭州可用区J）

**Comment by suxichuan** [comment]
我也用这个kuma还挺好用的
